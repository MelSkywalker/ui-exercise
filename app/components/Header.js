import React from 'react';
import GmailIcon from './svg/GmailIcon';
import MenuIcon from './svg/MenuIcon';

import './header.scss';

const Header = () => {
	return (
		<div className="header">
			<div className="left-container">
				<MenuIcon />
				<GmailIcon />
				<h1 className="title">Gmail</h1>
			</div>
			<div className="right-container">
			<div className="circle">
				<p className="first-letter">M</p>
			</div>
			</div>
		</div>
	)
}

export default Header;
