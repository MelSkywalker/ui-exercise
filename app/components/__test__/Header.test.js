import React from 'react';
import { shallow } from 'enzyme';

import Header from '../Header';

describe('components | Header', () => {
	it('it renders the component', () => {
		const component = shallow(<Header />);
		expect(component).toMatchSnapshot();
	});
});
