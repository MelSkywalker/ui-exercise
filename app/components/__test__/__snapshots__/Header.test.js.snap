// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | Header it renders the component 1`] = `
<div
  className="header"
>
  <div
    className="left-container"
  >
    <MenuIcon />
    <GmailIcon />
    <h1
      className="title"
    >
      Gmail
    </h1>
  </div>
  <div
    className="right-container"
  >
    <div
      className="circle"
    >
      <p
        className="first-letter"
      >
        M
      </p>
    </div>
  </div>
</div>
`;
